src/devices/openxrheadset/OpenXrInterface.cpp

```diff
@@ -211,6 +221,23 @@ bool OpenXrInterface::prepareXrInstance()
             return false;
     }
 
+    if (m_pimpl->hand_tracking_supported)
+    {
+        XrResult result = xrGetInstanceProcAddr(m_pimpl->instance, "xrCreateHandTrackerEXT",
+            (PFN_xrVoidFunction*)&m_pimpl->pfn_xrCreateHandTrackerEXT);
+        if (!m_pimpl->checkXrOutput(result, "Failed to load xrCreateHandTrackerEXT function pointer"))
+            return false;
+        result = xrGetInstanceProcAddr(m_pimpl->instance, "xrLocateHandJointsEXT",
+            (PFN_xrVoidFunction*)&m_pimpl->pfn_xrLocateHandJointsEXT);
+        if (!m_pimpl->checkXrOutput(result, "Failed to load xrLocateHandJointsEXT function pointer"))
+            return false;
+        if (m_pimpl->pfn_xrCreateHandTrackerEXT == nullptr || m_pimpl->pfn_xrLocateHandJointsEXT == nullptr)
+        {
+            yCError(OPENXRHEADSET) << "Failed to load hand tracking function pointers!";
+            return false;
+        }
+    }
```

Comment:
Here instead of returning ``false``, you could print the error and set ``m_pimpl->hand_tracking_supported`` to false. This could fix the test, where the extension is supported by the runtime, but it cannot be used since there is no headset connected.